kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/tasks/GenerateEmbeddedJreTask.kt

```diff
@@ -0,0 +1,62 @@
+package godot.gradle.tasks
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import java.util.*
+
+open class GenerateEmbeddedJreTask : DefaultTask() {
+
+    @Input
+    var modules: Array<String> = arrayOf(
+        "java.base",
+        "java.logging",
+    )
+
+    @Input
+    var outputDir: String = "jvm/jre-${getArch()}-${getOs()}"
```

Comment:
Wonder if we should declare the jvm directory path in the tool-common directory, to avoid hardcoding it here.